src/pages/AllocationPage/AllocationPage.tsx

```diff
@@ -118,6 +119,18 @@ const useStyles = makeStyles(theme => ({
     marginLeft: theme.spacing(1),
     padding: '1px',
   },
+  learnMore: {
+    margin: '0.5em',
+  },
+  icon: {
```

Comment:
this doesn't appear to be used in this file

Reply:
@MarcogodoyV just to be clear, i was only referring to the `icon` styles, so you didn't have to remove `learnMore` as well. but if it still looks good, that's ok too :)